nifi-commons/nifi-utils/src/main/java/org/apache/nifi/processor/util/StandardValidators.java

```diff
@@ -980,4 +980,25 @@ public ValidationResult validate(final String subject, final String value, final
             return new ValidationResult.Builder().subject(subject).input(value).explanation(reason).valid(reason == null).build();
         }
     }
+
+    public static final Validator SINGLE_CHAR_VALIDATOR = (subject, input, context) -> {
```

Comment:
I've never worked in this file, but at a glance it seems the intent it to keep all the static validators together in one section at the top of the file, so would you mind relocating this?

Reply:
I moved the method